api/routes/orders.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express')
+const router = express.Router()
+
+const {models} = require('../../db/')
+
+router.get('/', (req, res, next) => {
+  models.Orders.findAll({
+    include: [{model: models.LineItems, include: models.Products }],
+    order: [['createdAt', 'DESC']]
+  })
+    .then( response => res.send(response))
+})
+
+router.get('/cart', async (req, res, next) => {
+  if (!req.user){
```

Comment:
for guest user, `req.user` doesn't exisit and empty object is returned, instead a new cart should be created and sent.

Reply:
for guest user, create and send `order`